eslint/backend.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+    extends: [
+        "./index.js",
+        "plugin:@typescript-eslint/recommended",
+        "prettier/@typescript-eslint"
```

Comment:
Curious if we should use "plugin:prettier/recommended" as well?  We've used it on Engage to displays prettier errors as ESLint errors.